stock_picking_partner_note/models/stock_picking.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    note = fields.Html(compute="_compute_note", store=True)
+
+    @api.depends("partner_id")
+    def _compute_note(self):
+        for picking in self:
+            picking_type_note_type_ids = picking.picking_type_id.partner_note_type_ids
+            picking_notes = picking.partner_id.stock_picking_note_ids.filtered(
+                lambda n: n.note_type_id in picking_type_note_type_ids
```

Comment:
```suggestion
                lambda n: n.active and n.note_type_id in picking_type_note_type_ids
```
the result must be the same even if the active_test flag is set